sunpy/database/tests/test_database.py

```diff
@@ -440,6 +440,17 @@ def test_add_entry_from_hek_qr(database):
     # *something*
     assert len(database) > 1
 
+@pytest.mark.remote_data
+def test_download_from_hek_query_result(database, hek_qr, tmpdir):
+    assert len(database) == 0
+    database.download_from_hek_query_result(
+        hek_qr, path=str(tmpdir.join('{file}.fits')))
+    fits_pattern = str(tmpdir.join('*.fits'))
+    num_of_fits_headers = sum(
+        len(fits.get_header(file)) for file in glob.glob(fits_pattern))
```

Comment:
Did you need to sum to get the number of files?

Reply:
No I don't need to calculate the sum, and it's not necessarily needed for the method.  

Actually the same goes for lines 448 - 451. They are not necessary to test this particular method however is consistent with the test cases written for other methods. 

Hence added them to be consistent with the rest of the codebase.

Reply:
That's fine.